rasa/cli/data.py

```diff
@@ -531,6 +536,9 @@ def _warn_about_manual_forms_migration() -> None:
 
 
 def _assert_nlu_pipeline_given(config: Dict, policy_names: List[Text]) -> None:
+    from rasa.core.policies.fallback import FallbackPolicy
```

Comment:
Good timing with the review 🚀 I'm currently working on [this issue](https://github.com/RasaHQ/rasa/issues/8128) and I would even go so far to drop the policy imports completely in favor of using constants in `rasa.shared.core` for them. For `test_rasa_data` this means:

* current `main`: ~140 seconds
* this branch: ~85 seconds
* this branch + dropped policy imports in `rasa.core.config` and `rasa.cli.data`: ~35 seconds

In my opinion up to you if you want to do the extra step right away or I do a followup PR

Reply:
yes that is a good point and a great idea :+1: added it to this PR